src/plugins/intel_cpu/src/emitters/snippets/riscv64/cpu_generator.cpp

```diff
@@ -63,7 +63,9 @@ class jit_snippet : public ov::intel_cpu::riscv64::jit_generator_t {
 
     ~jit_snippet() override = default;
 
-    jit_snippet() = default;
+    static constexpr size_t max_code_size_bytes = 64 * 1024;
```

Comment:
Could you please provide a comment on why we have to set a custom max code size?